python/konduit/base_inference.py

```diff
@@ -60,89 +64,85 @@ class SavedModelConfig(object):
         'saveModelOutputOrder': 'table',
     }
 
-    def __init__(self, saved_model_path=None, model_tag=None, signature_key=None, saved_model_input_order=None, save_model_output_order=None
-                 ):
+    def __init__(self
+            , saved_model_path=None
+            , model_tag=None
+            , signature_key=None
+            , saved_model_input_order=None
+            , save_model_output_order=None
+            ):
         self.__saved_model_path = saved_model_path
         self.__model_tag = model_tag
         self.__signature_key = signature_key
         self.__saved_model_input_order = saved_model_input_order
         self.__save_model_output_order = save_model_output_order
-
+    
     def _get_saved_model_path(self):
         return self.__saved_model_path
-
     def _set_saved_model_path(self, value):
         if not isinstance(value, str):
             raise TypeError("savedModelPath must be str")
         self.__saved_model_path = value
     saved_model_path = property(_get_saved_model_path, _set_saved_model_path)
-
+    
     def _get_model_tag(self):
         return self.__model_tag
-
     def _set_model_tag(self, value):
         if not isinstance(value, str):
             raise TypeError("modelTag must be str")
         self.__model_tag = value
     model_tag = property(_get_model_tag, _set_model_tag)
-
+    
     def _get_signature_key(self):
         return self.__signature_key
-
     def _set_signature_key(self, value):
         if not isinstance(value, str):
             raise TypeError("signatureKey must be str")
         self.__signature_key = value
     signature_key = property(_get_signature_key, _set_signature_key)
-
+    
     def _get_saved_model_input_order(self):
         return self.__saved_model_input_order
-
     def _set_saved_model_input_order(self, value):
-        if not isinstance(value, list) and not isinstance(value, ListWrapper):
+        if not isinstance(value, list) and not isinstance(value,ListWrapper):
             raise TypeError("savedModelInputOrder must be list")
         if not all(isinstance(i, str) for i in value):
             raise TypeError("savedModelInputOrder list valeus must be str")
```

Comment:
There's this `valeus` typo at several places.

Reply:
@maxpumperla let's use this as an excuse to update the code gen to make sure we have proper output values. Thanks @ShamsUlAzeem